examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -245,12 +244,51 @@ class MeshLoadersApp : public ApplicationBase
             write.info = &info;
             logicalDevice->updateDescriptorSets(1u, &write, 0u, nullptr);
         }
+
         {
+            for (size_t i = 0ull; i < meshRaw->getMeshBuffers().size(); ++i)
+            {
+                auto& meshBuffer = meshRaw->getMeshBuffers().begin()[i];
+
+                for (size_t i = 0ull; i < nbl::asset::SBlendParams::MAX_COLOR_ATTACHMENT_COUNT; i++)
+                    meshBuffer->getPipeline()->getBlendParams().blendParams[i].attachmentEnabled = (i == 0ull);
+
+                meshBuffer->getPipeline()->getRasterizationParams().frontFaceIsCCW = false;
+
+                // TODO: This should come in set by the loader
+                asset::ICPUDescriptorSet* ds = meshBuffer->getAttachedDescriptorSet();
+                for (uint32_t j = 0u; j < ds->getMaxDescriptorBindingIndex(); ++j)
+                {
+                    const size_t arrayElementCount = ds->getDescriptors(j).size();
+                    for (size_t k = 0ull; k < arrayElementCount; ++k)
+                    {
+                        auto& currentDescriptor = ds->getDescriptors(j).begin()[k].desc;
+                        if (currentDescriptor->getTypeCategory() == asset::IDescriptor::EC_IMAGE)
+                        {
+                            asset::ICPUImageView* cpuImageView = static_cast<asset::ICPUImageView*>(currentDescriptor.get());
+                            const asset::E_FORMAT cpuImageViewFormat = cpuImageView->getCreationParameters().format;
+
+                            asset::IImage::E_ASPECT_FLAGS aspectFlags = asset::IImage::EAF_COLOR_BIT;
+                            if (isDepthOrStencilFormat(cpuImageViewFormat) && !isDepthOnlyFormat(cpuImageViewFormat))
+                            {
+                                if (isStencilOnlyFormat(cpuImageViewFormat))
+                                    aspectFlags = asset::IImage::EAF_STENCIL_BIT;
+                                else
+                                    aspectFlags = asset::IImage::EAF_DEPTH_BIT;
+                            }
+
+                            cpuImageView->setAspectFlags(aspectFlags);
+                        }
+                    }
+                }
+            }
```

Comment:
please move these fixes to the loader

why are you setting aspect flags? I thought @Erfan-Ahmadi fixed that in the loaders.

Reply:
It seemed It's not merged yet. the work is on CommonAPI branch.

Reply:
Resolved, needed some basic changes in `CGraphicsPipelineLoaderMTL` after the fact